tests/lcs/core/interface/snapshot.livecodescript

```diff
@@ -0,0 +1,75 @@
+script "CoreInterfaceSnapshot"
+/*
+Copyright (C) 2015 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on _TestImportScreenSnapshotFromRectHandler
+    import snapshot from rectangle 100,100,500,400
+    
+    TestAssert "long id of new image is long id of it",  the long id of it is the long id of the last image
+    TestAssertBroken "long id of image is it", it is the long id of the last image
+end _TestImportScreenSnapshotFromRectHandler
+
+on TestImportScreenSnapshotFromRectNoUI
+    if the environment is not "command line" then 
+        return "SKIP TEST"
+    end if  
+    TestAssertThrow "should now throw an error", "_TestImportScreenSnapshotFromRectHandler",\
```

Comment:
@livecodealex I'm having trouble with this test and the one below because it isn't throwing the error on command line. I picked it up because the standalone test runner PR runs tests on travis + linux as -ui but the tests also fail for me locally on mac:
```
FAIL:  tests/lcs/core/interface/snapshot: TestImportScreenSnapshotFromRectNoUI [1/2]
not ok - should now throw an error

PASS:  tests/lcs/core/interface/snapshot: TestImportScreenSnapshotFromRectUI [1/1]
FAIL:  tests/lcs/core/interface/snapshot: TestImportSnapshotFromStack [1/2]
not ok - should now throw an error

XFAIL: tests/lcs/core/interface/snapshot: TestImportSnapshotFromObject [2/3]
XFAIL: tests/lcs/core/interface/snapshot: TestImportSnapshotFromRectOfObject [2/3]
```